data/json/items/gun/84x246mm.json

```diff
@@ -21,6 +21,21 @@
     "clip_size": 1,
     "reload": 300,
     "loudness": 200,
-    "valid_mod_locations": [ [ "accessories", 4 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "sights", 1 ], [ "sling", 1 ] ]
+    "valid_mod_locations": [ [ "accessories", 4 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "sights", 1 ] ]
+  },
+  {
+    "id": "AT4",
+    "type": "GUN",
+	  "copy-from": "LAW",
+    "name": "AT4",
+    "description": "Mil-Spec Rocket Launcher. An 84-mm unguided, portable, single-shot recoilless smoothbore weapon used primarily by the US Military.",
+	  "flags": [ "NO_UNLOAD", "BACKBLAST", "NEVER_JAMS", "TRADER_AVOID", "FIRE_TWOHAND", "NO_REPAIR" ],
```

Comment:
If you want to add flags to the ones you inherited from `copy-from`, use 
`"extend": { "flags": [ "FIRE_TWOHAND", "NO_REPAIR" ] }`
instead.